src/vs/workbench/contrib/preferences/browser/settingsWidgets.ts

```diff
@@ -322,9 +316,8 @@ abstract class AbstractListSettingWidget<TDataItem extends object> extends Dispo
 			this.listElement.appendChild(header);
 		}
 
-		this.model.items
-			.map((item, i) => this.renderDataOrEditItem(item, i, focused))
-			.forEach(itemElement => this.listElement.appendChild(itemElement));
+		this.rowElements = this.model.items.map((item, i) => this.renderDataOrEditItem(item, i, focused));
```

Comment:
@roblourens is it better to store these elements in an array like this? Or should I use `listElement.children`? I'm not sure what the best practices are regarding this. 🙈

Reply:
I think this is fine!